examples/derived.ts

```diff
@@ -1,9 +1,18 @@
 import * as C from "http://localhost:5646/@local/mod.ts"
 import * as U from "http://localhost:5646/@local/util/mod.ts"
 
-const ids = C.entryRead(C.polkadot)("Paras", "Parachains", [])
-  .access("value")
-  .as<number[]>()
+// TODO: uncomment these lines to run a single effect upon solving `count` in zones
+// or effects/rpc.ts#discardCheck
+// const ids = C.entryRead(C.polkadot)("Paras", "Parachains", [])
+//   .access("value")
+//   .as<number[]>()
+
+const ids = U.throwIfError(
+  await C.entryRead(C.polkadot)("Paras", "Parachains", [])
+    .access("value")
+    .as<number[]>()
+    .run(),
+)
```

Comment:
I re-wrote this because there is an issue with the client reference counting.
After
```
 const ids = C.entryRead(C.polkadot)("Paras", "Parachains", [])
   .access("value")
   .as<number[]>()
```
The client reference count reaches `1` and it's discarded in https://github.com/paritytech/capi/blob/7edf6a2c26fd55b49f68108cd989f1f73134061d/effects/rpc.ts#L93-L102

The above invokes the `proxyProvider.release` that removes the `WebSocket` listener.
As a result the next call `C.Z.each(...)` successfully creates a new `WebSocket` but there is not listener for the incoming messages.